service/project.go

```diff
@@ -350,6 +350,23 @@ func (uis *UIServer) modifyProject(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	if !utility.FromBoolPtr(origProjectRef.Enabled) && responseRef.Enabled {
+		settings, err := evergreen.GetConfig()
+		if err != nil {
+			uis.LoggedError(w, r, http.StatusInternalServerError, err)
+			return
+		}
+		err, _ = model.ValidateProjectCreation(responseRef.Identifier, settings, &model.ProjectRef{
```

Comment:
this comment may have gotten missed but we really should never return error before value; can ValidateProjectCreation be updated?

Reply:
done, and will update return values in a later ticket